filecoin-proofs/src/bin/phase2.rs

```diff
@@ -452,13 +455,18 @@ fn create_initial_params<Tree: 'static + MerkleTreeTrait>(
         info!("finished writing large params to file");
     }
 
+    // TODO: add conversion from large to small params to phase2 crate, then write initial params as
+    // small-raw.
+    /*
+    let small_path = params_filename(proof, hasher, sector_size, &head, 0, ParamSize::Small, true);
```

Comment:
I assume this is something we should do before the next trusted setup. Can you please create an issue to track this? (Fine to link to this TODO.).

Reply:
Ok, I made issues in rust-fil-proofs and phase2.